_positions/Innovation-Specialist-GS-13.md

```diff
@@ -0,0 +1,88 @@
+##Innovation Specialist, GS-0301-13
```

Comment:
@rooneywp To ensure that this position gets added and displayed correctly, please add the lines that are at the start of the other position descriptions (ex: lines 1 through 7 of https://github.com/18F/joining-18f/edit/18f-pages/_positions/content-designer.md). If you'd like to pair on that, let me know.